routes/auth.js

```diff
@@ -11,6 +11,11 @@ function createToken(user) {
     return jwt.sign(_.omit(user, 'password'), config.secretKey, { expiresIn: 60*60*5 });
 }
 
+function validateEmail(email) {
+    const re = /\S+@\S+\.\S+/;
```

Comment:
this unfortunately does validate email addresses at all. FYI, [here's](http://www.regular-expressions.info/email.html) a decent article about email regex. There are lots other papers available online, if you are interested.
so, either go with one of the creditable open source email validators or use the regex suggested by RFC standards